lnrpc/autopilotrpc/autopilot.proto

```diff
@@ -54,3 +60,9 @@ message QueryScoresResponse {
 
     repeated HeuristicResult results = 1 [json_name = "results"];
 }
+
+message SetScoresRequest{
+    map<string, int64> scores= 1 [json_name = "scores"];
```

Comment:
Nit: add space between `scores=`.

Reply:
Needs comment, specifically to explain what the keys and values should be

Why use `int64` and not `double`?

Reply:
Good catch, this should be `double`. It initially was using integers, but was changed to floats. The RPC I forgot...

Reply:
Fixed.